Formula/yelp-tools.rb

```diff
@@ -19,11 +19,9 @@ class YelpTools < Formula
   end
 
   depends_on "gettext" => :build
-  depends_on "itstool" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
-  depends_on "gtk+3"
```

Comment:
Looking through commit history, `gtk+3` looks to be from `yelp-xsl` to run post-install (https://github.com/Homebrew/homebrew-core/commit/6335dea3550007c7e2346ccd7234b5017b9f832e)

However, `yelp-xsl` switched to inline SVG icons since 3.28.0 -- https://gitlab.gnome.org/GNOME/yelp-xsl/-/commit/1993f9a2dec7fac104fc09bc84b1ce5dbb0b76e9